src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/AbstractDedupJobScheduler.java

```diff
@@ -86,27 +86,40 @@ public TimeJobInfo reserveSingle(final String tube, final long ttrInMillis) {
     }
 
     public long getReadyJobCount(String tube) {
-        return getJobCount(getReadyQueue(tube));
+        final String queue = getReadyQueue(tube);
+        final long now = Instant.now().getMillis();
+        return rdbi.withHandle(new Callback<Long>() {
+            @Override
+            public Long run(Handle handle) {
+                return handle.jedis().zcount(queue, 0, now);
```

Comment:
That is the same logic we use in reservation (https://github.com/lithiumtech/rdbi/blob/master/src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/StateDedupedJobSchedulerDAO.java#L54) or even in the same class, in peaking method (https://github.com/lithiumtech/rdbi/blob/master/src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/AbstractDedupJobScheduler.java#L101).